torch/csrc/jit/codegen/cuda/kernel_db/test/test_nvfuser_kernel_db_write.cpp

```diff
@@ -0,0 +1,77 @@
+#if defined(USE_CUDA)
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <torch/torch.h>
+
+#include <torch/csrc/jit/codegen/cuda/kernel_db/kernel_db.h>
+#include <torch/csrc/jit/codegen/cuda/kernel_db/utils.h>
+#include <torch/csrc/jit/codegen/cuda/test/test_gpu_validator.h>
+#include <torch/csrc/jit/codegen/cuda/test/test_utils.h>
+
+// Tests go in torch::jit
+namespace torch {
+namespace jit {
+
+namespace Nvf = torch::jit::fuser::cuda;
+
+// RUN CMD: bin/test_jit --gtest_filter="NVFuserTest*KernelDb_Write*"
+TEST_F(NVFuserTest, KernelDb_Write_CUDA) {
+  // Setup the test db
+  fs::path test_data =
+      fs::path(__FILE__).parent_path() / "test_data/kernel_db_for_query_test";
```

Comment:
shouldn't the folder path here be `kernel_db_for_write_test` instead of `query`?